README.md

```diff
@@ -0,0 +1,47 @@
+# Remote-Settings-Client (Project Ubique)
+
+A Remote-Settings Client built using Rust to read collection data.
+
+- Read-Only
+- Customizable Signature Verification
+- Cross-Platform
+- Robust
+- Rust!
+
+## Example
+
+Below example uses [log](https://docs.rs/log/0.4.8/log/), [env_logger](https://docs.rs/env_logger/0.7.1/env_logger/), and [Tokio](https://tokio.rs) and utilizes some optional features, so your `Cargo.toml` could look like this:
```

Comment:
I think it's better not to mention version in URLs `https://docs.rs/log/`, `https://docs.rs/env_logger/`